compiler-cli/src/add.rs

```diff
@@ -36,18 +22,34 @@ pub fn command(to_add: String, dev: bool) -> Result<()> {
             err: Some(e.to_string()),
         })?;
 
-    // Insert the new dep
-    #[allow(clippy::indexing_slicing)]
-    if dev {
-        toml["dev-dependencies"][&to_add] = toml_edit::value(range);
-    } else {
-        toml["dependencies"][&to_add] = toml_edit::value(range);
-    };
+    // Insert the new deps
+    for package_to_add in packages {
+        // Pull the selected version out of the new manifest so we know what it is
+        let version = &manifest
+            .packages
+            .iter()
```

Comment:
Rust complained about `manifest.packages` being moved until I changed `.into_iter()` to `.iter()` and borrowed `manifest`, perhaps there is a better solution to this?

Reply:
This is perfect 👍